keyboards/handwired/videowriter/info.json

```
@@ -0,0 +1,20 @@
+{
+    "keyboard_name": "videowriter",
+    "url": "mailto:quartz64@gmail.com",
+    "maintainer": "quartz64",
```

Again, this should be your GitHub username.

```suggestion
    "maintainer": "DmNosachev",
```